### PR TITLE
Update development instructions

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

**Comment by electrum** [comment]
Please try out the new development instructions.

**Review by dain — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -40,25 +40,40 @@ Install gems for site:
...
 
 You can run the server on your computer and then verify the pages render
-correctly on a phone.  By default the server only binds to localhost, so it is
-not accessible on your local network.  Add `--host 0.0.0.0` to the `bundle exec`
-command to have the server bind to the external IP addresses on your computer.
+correctly on a phone. There are two ways to do that:
+
+* Access the IP address of your computer on your local network from your phone.
+
+* Run `netlify dev --live` to create a publically accessible tunnel that can
```
Comment (bitsondatadev) [inline README.md:69]: I got this error when runing --live
```
◈ Error: no siteId defined, did you forget to run netlify init or netlify link?
```

Should we use some sort of shared site or create our own account?

Comment (bitsondatadev) [reply README.md:69]: It was pretty straightforward to just create my own so I don't think we need to document it here perse...just mention that you'll need to create and login or `netlify link` your site/account.

**Comment by bitsondatadev** [comment]
We should also consider adding this to .gitignore?
```
.netlify
```

File: README.md (hunk 1 above)
Comment (electrum) [reply README.md:69]: Can you try linking it to the existing site? Or so you have to have a Netlify login of your own?

**Comment by electrum** [comment]
@bitsondatadev It's added to `.gitignore` in this PR

**Pull request merged**